### PR TITLE
2764 Fixed expansion panel indicator

### LINKED ISSUE #50: Center-Align Chevrons in Expansion Panel
#### Describe the bug
Chevrons should be center aligned in the Tree Structure example.
This might be a bigger issue; should this apply to all chevrons
Should this be a themes change?  Might need to understand why these icons aren't centered in the first place.

#### What is the expected behavior?
Chevron should be center-aligned when opened or closed. 

#### What are the steps to reproduce?
Go to tree structure demo.
Look at the chevrons.

#### Screenshots or links to minimum reproduction example
140258600-de3623fa-bdda-4049-8356-391f2d4879d0

#### Environment

Window Chrome
		
#### Anything else to add?
Nope

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes #50  

#### Changes proposed in this Pull Request:
- Fixed expansion panel chevron

#### Screenshots:
**Desktop**
Screenshot 2022-01-06 at 1 00 12 PM

**Mobile**
Screenshot 2022-01-06 at 1 01 07 PM

#### To Test:
- yarn start
- Open Surfaces under Material Components 
- Scroll down to Accordion

### CODE REVIEW

**Comment by emclaug2** [comment]
Code looks good, just have to run it.  Can you update the CHANGELOG?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## v6.3.1 (Not yet published)
 
+### Fixed
+-   Fixed chevron icon in `<mat-expansion-panel-header>`.
+
+## v6.3.1 (Not yet published)
```
Comment (emclaug2) [inline CHANGELOG.md:8]: Duplicate version here.

Also "fixed chevron" is a little vague; how about:

"Fixed non-center aligned chevron in `<mat-expansion-panel-header>`.

Comment (emclaug2) [reply CHANGELOG.md:8]: Also make sure to merge dev into your current branch; there may be a change conflict now.

**Review by jeffvg — APPROVED** [review]
looks good

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 # Changelog
 
 ## v6.3.1 (Not yet published)
+
+### Fixed
```
Comment (emclaug2) [inline CHANGELOG.md:5]: Fixed #1

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+
+-   Fixed non-center aligned chevron in <mat-expansion-panel-header>
 ### Changed
 
 -   Changed default font-size of `.mat-tooltip` text to `12px`.
+
```
Comment (emclaug2) [inline CHANGELOG.md:11]: Fixed #2, please remove the duplicate 'fixed' section

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**